java/ql/src/experimental/Security/CWE/CWE-451/clickjacking.qhelp

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>
+Web sites that do not specify the <code>X-Frame-Options</code> or specify <code>SAMEORIGIN</code> HTTP header may be vulnerable to UI
+redress attacks ("clickjacking"). In these attacks, the vulnerable site is loaded in a frame on
+an attacker-controlled site which uses opaque or transparent layers to trick the user into
+unintentionally clicking a button or link on the vulnerable site.
+</p>
+
+</overview>
+<recommendation>
+
+<p>
+Set the <code>X-Frame-Options</code> HTTP header to <code>DENY</code>, to instruct web browsers to
+block attempts to load the site in a frame. Alternatively, if framing is needed in certain
+circumstances, specify <code>ALLOW FROM: ...</code> to limit the ability
```

Comment:
`ALLOW FROM` is obsolete according to the Mozilla [documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Frame-Options).